documentation/building/official-build-instructions.md

```diff
@@ -5,13 +5,20 @@ WARNING: These instructions will only work internally at Microsoft.
 
 To kick off an official build, go to this build definition: https://dev.azure.com/dnceng/internal/_build?definitionId=528.
 
-This signs and publishes dotnet-dump, dotnet-sos, dotnet-trace, dotnet-counters and Microsoft.Diagnostic.TestHelpers packages to https://dotnetfeed.blob.core.windows.net/dotnet-core feed.
+This signs and publishes the following packages to the dotnet feed (currently https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet5/nuget/v3/index.json):
+ - dotnet-dump
+ - dotnet-sos
+ - dotnet-trace
+ - dotnet-counters
+ - dotnet-monitor
+ - Microsoft.Diagnostic.TestHelpers
+ - Microsoft.Diagnostics.NETCore.Client
 
 To release the latest tools:
 
-1) Merge all the commits for this release from the master branch to the release/3.0 branch.
-2) Kick off an official build from the release/3.0 branch.
-3) Change all the package version references in the documentation folder to this official build's package version. Commit/merge to master and release/3.0.
-4) Download the above packages from the successful official build under "Artifacts" -> "packages".
+1) Merge all the commits for this release from the master branch to the release/stable branch.
+2) Kick off an official build from the release/stable branch.
+3) Change all the package version references in the documentation folder to this official build's package version to maintain the docs up to date. Merge this to both the release and the master branches.
```

Comment:
Not sure we need this step anymore. This was when the docs had preview versions spelled out in the `dotnet tool install -g` command.  I guess there is still the message from the tool install that contains the version:

```
Tool 'dotnet-dump' (version '3.0.47001') was successfully installed.
```

I'm not sure how important it is to change them.  I have forgot to do it the last 3 or 4 releases.